ki.js

```diff
@@ -10,7 +10,7 @@
    * a = selector, dom element or function
    */
   function i(a) {
-    c.push.apply(this, a && a.nodeType ? [a] : '' + a === a ? b.querySelectorAll(a) : e)
+    c.push.apply(this, a && a.nodeType ? [a] : '' + a === a ? b.querySelectorAll(a) : a)
```

Comment:
This change does not make sense. For everything that's neither node nor string, we should either call apply with undefined or an empty array. Since we already got e set as undefined, it is fine to use it.

If you called it with `[0]`, it would seem as a selection of the number zero, instead of an empty selection.

Reply:
`c._` would work too, then you don't need `e` at all anymore. I wonder if it saves bytes this way.

Reply:
that's what i thought @atk thanks for confirming that

Reply:
@tarjei read the comment above from @atk

Reply:
@dciccale ok. As I understand @atk's comment, I should just let the code continues as it is :)